### PR TITLE
Consolidating initializing message to a single line

### PR DESCRIPTION
**Pull request by AkashGutha**
### Requirements

Consolidating the logging info into a single line.

### Description of the Change

Still a work in progress. So, the idea is to implement a combined logger.
is that the aim of #203

### Alternate Designs

If not we can just remove one of the log lines

### Benefits

Having a combined logger will make things simpler going forward.

### Possible Drawbacks

I possibly don't see any.

### Applicable Issues

#203

### CODE REVIEW

**Comment by AkashGutha** [comment]
Messed up something in the original branch. So, I had to redo the pull again.

**Comment by StanleyGoldman** [comment]
Hey @AkashGutha normally we don't close and re-open pull requests to make changes. The history of what I asked you to change and why I asked you to change it is lost.

Normally, you push additional commits to the same branch or you can force push to the same branch.

File: src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs [hunk 1]
```diff
@@ -34,10 +34,11 @@ private static void Initialize()
 
             // this will initialize ApplicationManager and Environment if they haven't yet
             var logPath = Environment.LogPath;
+            var logInfo = "";
```
Comment (shana) [inline src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs]: If you're only using this variable inside the block, you shouldn't declare outside the block

File: src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs [hunk 2]
```diff
@@ -53,7 +54,8 @@ private static void Initialize()
                     Logging.Error(ex, "Error rotating log files");
                 }
 
-                Debug.Log("Initialized GitHub for Unity log file: " + logPath);
+                logInfo += " and Unity log file: " + logPath;
+                Debug.Log(logInfo);
```
Comment (shana) [inline src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs]: There's really no need to have a separate variable to store the message before outputting, you're overcomplicating things here.

Comment (AkashGutha) [reply src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs]: Okay, I'll just redo that

**Comment by AkashGutha** [comment]
@shana can you check the changes. this must be fine.

**Comment by StanleyGoldman** [comment]
Hey @AkashGutha you have to be a bit more considerate sometimes. I believe @shana is traveling.

It's okay though, I got it from here.
I tweaked the message a bit to use two lines.

img

I think it looks pretty good!
Thanks for your help.

We look forward to your next pull request.

**Comment by StanleyGoldman** [comment]
img

**Pull request merged**